### PR TITLE
Feature update fapi

### PR DESCRIPTION
**Pull request by incorbador**

### CODE REVIEW

File: packages/web-core/src/services/ConnectService.ts [hunk 1]
```diff
@@ -121,16 +121,16 @@ export class ConnectService {
   }
 
   async loginInit(abortController: AbortController): Promise<Result<ConnectLoginInitData, CorbadoError>> {
-    const existingProcess = ConnectProcess.loadFromStorage(this.#projectId);
-    if (existingProcess?.isValid()) {
-      log.debug('process exists, preparing api clients');
-      this.#setApisV2(existingProcess);
-    }
+    // const existingProcess = ConnectProcess.loadFromStorage(this.#projectId);
```
Comment (incorbador) [inline packages/web-core/src/services/ConnectService.ts:124]: this effectively makes sure that we get a new process every time when a login is initiated

File: packages/web-core/src/services/ConnectService.ts [hunk 2]
```diff
@@ -314,6 +309,7 @@ export class ConnectService {
   async startAppend(
     appendTokenValue: string,
     abortController?: AbortController,
+    initiatedByUser?: boolean,
```
Comment (incorbador) [inline packages/web-core/src/services/ConnectService.ts:312]: this indicates if the append was started by the user (i.e. from passkey-list) or if it was offered by the component (i.e. after a login)

File: packages/web-core/src/services/ConnectService.ts [hunk 3]
```diff
@@ -516,10 +512,10 @@ export class ConnectService {
         isUserVerifyingPlatformAuthenticatorAvailable: canUsePasskeys,
         isConditionalMediationAvailable: canUseConditionalUI,
         clientEnvHandle: maybeClientHandle ?? undefined,
+        visitorId: currentVisitorId,
```
Comment (incorbador) [inline packages/web-core/src/services/ConnectService.ts:515]: For the backend it' easier to handle if visitorId is part of clientinformation (it is also more similar to clientEnvHandle then)

File: packages/web-core/src/services/ConnectService.ts (hunk 1 above)
Comment (Dopeamin) [reply packages/web-core/src/services/ConnectService.ts:124]: yes makes sense.

**Review by Dopeamin — APPROVED** [review]
All good, all makes clear sense !

**Pull request merged**